src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -291,6 +291,18 @@ public bool EnableTSpline
             }
         }
 
+        /// <summary>
+        /// Indicates whether experimental search mode is turned on.
+        /// </summary>
+        private bool ExperimentalSearch
+        {
+            get { return model.SearchModel.experimentalSearch; }
+            set
+            {
+                model.SearchModel.experimentalSearch = value;
```

Comment:
I guess in this case it's not such a big deal, but you usually would not modify a field from another object - I think instead we usually use a property, in this case I think it makes sense to try and hide this field from external consumers - we don't want anyone to use it - in an extension for example.